src/temporary_home/ReactiveSocket.cpp

```diff
@@ -138,6 +145,25 @@ void ReactiveSocket::requestStream(
       std::move(request), std::move(responseSink));
 }
 
+bool ReactiveSocket::requestStream(
+    Payload request,
+    yarpl::Reference<yarpl::flowable::Subscriber<Payload>> responseSink,
+    std::string streamName) {
+  debugCheckCorrectExecutor();
+  checkNotClosed();
+  auto streamId = connection_->getStreamId(streamName);
```

Comment:
Should this just be a CHECK()? Seems like programmer error, could also rename to `hasStreamId()`.